datasets/cord-19.yaml

```diff
@@ -12,7 +12,7 @@ Tags:
 License: Open (see license file for details)
 Resources:
   - Description: S3 bucket with CORD-19 dataset files
-    ARN: https://s3.console.aws.amazon.com/s3/buckets/ai2-semanticscholar-cord-19/
+    ARN: ai2-semanticscholar-cord-19
     Region: us-west-2
     Type: S3
```

Comment:
Resource type can be be CloudFront Distribution, DB Snapshot, S3 Bucket, or SNS Topic; just revise "S3" to "S3 Bucket"!

Reply:
Done!